portfolio/src/main/webapp/gallery.js

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const galleryImages = [
+    {
+        name: "cern.jpg",
+        caption: "My family at CERN"
+    },
+    {
+        name: "deer.jpg",
+        caption: "A deer looking through a window at my house"
+    },
+    {
+        name: "kungfu.gif",
+        caption: "My twin brother and I doing a kung fu form."
+    },
+    {
+        name: "mountains.jpg",
+        caption: "A picture of me at Mount Pilatus."
+    },
+    {
+        name: "dollar.jpg",
+        caption: "Me holding a Zimbabwean dollar I won in economics class."
+    }
+];
+
+shuffle(galleryImages);
+
+let HTMLImage;
```

Comment:
Like Nikhil pointed out in a different PR, be sure to keep a consistent capitalization scheme for your variables.  JavaScript typically uses camelCase for variables and functions, and CapCase for class names.

Reply:
Looks like I forgot to merge master back into this branch. Fixed and will do for the other branches as well.